tests/TestCase.php

```diff
@@ -92,7 +94,7 @@ protected function setUpDatabase(Application $app)
     protected function getPackageProviders($app)
     {
         return [
-            RayServiceProvider::class,
+            // RayServiceProvider::class,
```

Comment:
Why do we need to comment this?

Reply:
Tests don't work otherwise. I am not sure who introduced that service provider, but it's not in the composer dependencies list.